### PR TITLE
release 1.3.0

### PR DESCRIPTION
**Pull request by jeremad**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tankersdk-identity"
-version = "1.2.1"
+version = "1.3.0"
```
Comment (dmerejkowsky) [inline pyproject.toml]: 1.3.1, no ?

Comment (jeremad) [reply pyproject.toml]: 1.3.0 is the latest released, 1.3.1 is the future, I don't know what's the better option?

Comment (dmerejkowsky) [reply pyproject.toml]: never mind. Keep 1.3.1

**Comment by jeremad** [comment]
An approval please :angel: ?

**Pull request merged**